src/App/App.js

```diff
@@ -1,117 +1,114 @@
-import './App.css';
-// import searchIcon from '../icons/search.png';
-import { useState, useEffect } from 'react';
-import MovieDetails from '../MovieDetails/MovieDetails';
-import MoviesContainer from '../MoviesContainer/MoviesContainer';
-import Header from '../Header/Header';
-import { useNavigate, Routes, Route } from 'react-router-dom';
+import "./App.css";
+
+import { useState, useEffect } from "react";
+import MovieDetails from "../MovieDetails/MovieDetails";
+import MoviesContainer from "../MoviesContainer/MoviesContainer";
+import Header from "../Header/Header";
+import { useNavigate, Routes, Route } from "react-router-dom";
```

Comment:
Router was surprisingly intuitive! This greatly enhanced the speed and efficiency of our app.